prow/config.yaml

```diff
@@ -3844,6 +3844,26 @@ presubmits:
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
 
+  kubernetes-incubator/service-catalog:
+  - name: svc-cat-test
```

Comment:
for whatever reasons k8s jobs usually have `pull|ci|periodic-$name` /shrug

Reply:
More than happy to rename to match existing convention. What do the notations mean? Which should I choose?